dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -257,6 +259,7 @@ private static MethodSymbolReference constantToMethodReference(
         MethodSymbolReference.builder()
             .setSourceClassName(sourceClassName)
             .setMethodName(methodName)
+            .setInterfaceMethod(isInterfaceMethod)
```

Comment:
Could you do an instanceof check on `constantMethodRef` here instead of passing a boolean? Is that better or worse?

Note that if you did this, you could fold the `CONSTANT_Methodref` and `CONSTANT_InterfaceMethodref` cases above.

Reply:
Good idea. Committed change, with '// fall through' in the switch case.